apps/storefront/src/pages/OrderDetail/components/OrderAction.tsx

```diff
@@ -407,11 +418,7 @@ export default function OrderAction(props: OrderActionProps) {
     {
       header: b3Lang('orderDetail.payment'),
       key: 'payment',
-      subtitle: createAt
-        ? b3Lang('orderDetail.paidInFull', {
-            paidDate: displayFormat(createAt, true),
-          })
-        : '',
+      subtitle: getPaymentMessage() || '',
```

Comment:
With the suggestion above there is no need to validate here, just use what is returned from `getPaymentMessage()`
```suggestion
      subtitle: getPaymentMessage(),
```